### PR TITLE
Update completion.fish

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

File: completion.fish [hunk 1]
```diff
@@ -36,5 +36,6 @@ function __fish_complete_npm -d "Complete the commandline using npm's 'completio
     end
 end
 
+# flush out what ships with fish
 # flush out what ships with fish
```
Comment (henry-richard) [inline completion.fish:40]:
```suggestion
# flush out what ships with fish
# flush out what ships with fishing

```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Comment by wanda-carlson** [comment]
comment here

**Comment by wanda-carlson** [comment]
download

Look here

**Comment by wanda-carlson** [comment]
Or here
[download.jpeg](https://files.slack.com/files-pri/T0289BY1FK4-F0601699QS0/download.jpeg)

**Comment by wanda-carlson** [comment]
@henry-richard Can you look into this?

**Pull request merged**